docker/build_image

```diff
@@ -89,7 +89,7 @@ function recursiveDigest() {
 # Searches recursively for latest updated file and keeps this variable tracking that file.
 newestFile=$0
 function findNewest() {
-    lastModifiedFile=$(find $1 -f ! -path '*/node_modules/*' -printf "%T+ %p\n" | sort | tail -1 | cut -d ' ' -f2)
+    lastModifiedFile=$(find $1 -type f ! -path '*/node_modules/*' -printf "%T+ %p\n" | sort | tail -1 | cut -d ' ' -f2)
```

Comment:
just checking that this should still be `-f`?

Reply:
It should be `-type f`.  It still worked fine before but gave a noisy warning.